config/src/config/network_config.rs

```diff
@@ -47,6 +47,9 @@ pub struct NetworkConfig {
     #[serde(skip)]
     pub seed_peers: SeedPeersConfig,
     pub seed_peers_file: PathBuf,
+    // upstream_peers are the nodes that are upstream in the state sync protocol for this network
+    // This is empty for a validator network, since the upstream peers are populated via consensus
+    pub upstream_peers: Vec<PeerId>,
```

Comment:
I wonder do we actually need to add this parameter here?
cuz for PFN and Validator networks it's not going to be used
and most of VFN networks will just specify one validator
just curious why you decided to use this approach instead of introducing separate static UpstreamConfig?
we could even get rid of `gen_upstream_config` function and set it statically